# HG changeset patch
# User Shauren
# Date 1278492162 -7200
# Branch trunk
# Node ID f75327bb2fcd24a13b1e338e0077db99f6f0aca2
# Parent  9b469d0faddd59dc7ede51da158415be7bf30cab
Added drop statement before table rename
Because Aokromes wasnt happy.

diff -r 9b469d0faddd -r f75327bb2fcd sql/updates/8790_world_locales_item_set_names.sql
--- a/sql/updates/8790_world_locales_item_set_names.sql	Wed Jul 07 09:42:18 2010 +0200
+++ b/sql/updates/8790_world_locales_item_set_names.sql	Wed Jul 07 10:42:42 2010 +0200
@@ -1,1 +1,2 @@
+DROP TABLE IF EXISTS `locales_item_set_names`;
 RENAME TABLE `locales_item_set_name` TO `locales_item_set_names`;
